site/content/en/docs/Guides/feature-stages.md

```diff
@@ -24,17 +24,20 @@ that can be found in the [Helm configuration]({{< ref "/docs/Installation/Instal
 
 The current set of `alpha` and `beta` feature gates:
 
-| Feature Name                                                                                                          | Gate                     | Default  | Stage   | Since  |
-|-----------------------------------------------------------------------------------------------------------------------|--------------------------|----------|---------|--------|
-| [Custom resync period for FleetAutoscaler](https://github.com/googleforgames/agones/issues/1955)                      | `CustomFasSyncInterval`  | Enabled  | `Beta`  | 1.25.0 |
-| [Graceful Termination for GameServer SDK](https://github.com/googleforgames/agones/pull/2205)                         | `SDKGracefulTermination` | Enabled  | `Beta`  | 1.18.0 |
-| [GameServer state filtering on GameServerAllocations](https://github.com/googleforgames/agones/issues/1239)           | `StateAllocationFilter`  | Enabled  | `Beta`  | 1.26.0 |
-| [GameServer player capacity filtering on GameServerAllocations](https://github.com/googleforgames/agones/issues/1239) | `PlayerAllocationFilter` | Disabled | `Alpha` | 1.14.0 |
-| [Player Tracking]({{< ref "/docs/Guides/player-tracking.md" >}})                                                      | `PlayerTracking`         | Disabled | `Alpha` | 1.6.0  |
-| [Reset Metric Export on Fleet / Autoscaler deletion]({{% relref "./metrics.md#dropping-metric-labels" %}})            | `ResetMetricsOnDelete`   | Disabled | `Alpha` | 1.26.0 |
-| [GameServer Stable Network ID]({{% ref "/docs/Reference/gameserver.md#stable-network-id" %}})                         | `PodHostname`            | Disabled | `Alpha` | 1.29.0 |
-| [GameServer `SafeToEvict` API](https://github.com/googleforgames/agones/issues/2794)                                  | `SafeToEvict`            | Disabled | `Alpha` | 1.29.0 |
-| Example Gate (not in use)                                                                                             | `Example`                | Disabled | None    | 0.13.0 |
+{{% feature publishVersion="1.30.0" %}}
```

Comment:
You will want an `expiryVersion` block of this as well with the previous version in it.

Reply:
The merge should have brought in the `expiryVersion` block.